### DISCUSSION TITLE
Proxy server

### DISCUSSION THREAD

**Discussion by owenconnorz** [z-huang/music#205] [Ideas]
Hi dude I was thinking if you created a proxy I can definitely support the payments of it at all time you've given us a brilliant app just something in return😀

**Comment by z-huang** [comment]
Thanks for your suggestion and kindness. But I think this cannot be done.
1. This project focuses on creating a YTM client. Maintaining a proxy server is out of scope.
2. If YTM suggests things based on the geographical location of your IP (#202), users of the proxy server will have the same suggestions in the region of the proxy server. So, it won't be a good solution.
